packages/demand-api/src/payment/PaymentService.ts

```diff
@@ -22,11 +23,20 @@ export class PaymentService implements Payment {
   private apiV3 = 'v3'
 
   private http: Http
+  private providerApiVersion: ProviderVersion
```

Comment:
Let's try to get rid of this - it is Adyen specific, not API payments

Reply:
but then in PaymentBloc you need to pass version down to other methods as parameter and then that goes to the paymentService, I'll need to store that somewhere anyway